docs/running-locally.md

```diff
@@ -59,7 +59,9 @@ npm run build
 
 The built JavaScript is placed in the `scripts/dist` directory and is committed to source control (this part's easy to forget).
 
-If you're running JKAN inside docker, you should just be able to prefix these commands with `bundle exec jekyll`, e.g. `bundle exec jekyll npm run build`.
+If you're running JKAN inside docker, you should just be able to prefix these commands with `bundle exec jekyll`, e.g. `bundle exec jekyll npm run build` if you have jekyll-npm gem installed.
```

Comment:
Wow, did I really write `bundle exec jekyll npm run build`? I must have been very tired! I meant `docker compose exec jekyll npm run build` as you wrote below! 🤦🏻‍♂️